stages/03_process_data.py

```diff
@@ -0,0 +1,48 @@
+# Imports
+# -------
+import re
+import pathlib
+import pandas as pd
+
+extensions = ['ism']
+extensions_re = re.compile(r'\.(' + '|'.join(re.escape(ext) for ext in extensions) + r')$')
+files = filter( lambda item: item.is_file(), pathlib.Path('download').rglob('*'))
+
+brick_dir = pathlib.Path('brick')
+brick_dir.mkdir(exist_ok=True)
+
+for file in files:
```

Comment:
There are some `*.ism` files that are empty:

```shell
$ find download/ -type f -empty | wc -l
44
```

These should be skipped.

Reply:
Good catch, that's weird that some of ISM files are empty. I'll write a check in the script for an if statement.